README.md

```diff
@@ -1,106 +1,40 @@
-# nextstrain.org/monkeypox
+# Nextstrain repository for mpox virus
 
-This is the [Nextstrain](https://nextstrain.org) build for MPXV (mpox virus). Output from this build is visible at [nextstrain.org/monkeypox](https://nextstrain.org/monkeypox).
-The lineages within the recent mpox outbreaks in humans are defined in a separate [lineage-designation repository](https://github.com/mpxv-lineages/lineage-designation).
+This repository contains two workflows for the analysis of mpox virus (MPXV) data:
 
-## Software requirements
+- `ingest/` - Download data from GenBank, clean and curate it and upload it to S3
+- `phylogenetic/` - Make phylogenetic trees for nextstrain.org
 
-Follow the [standard installation instructions](https://docs.nextstrain.org/en/latest/install.html) for Nextstrain's suite of software tools.
+Each folder contains a README.md with more information.
 
-## Usage
+## CI
 
-### Provision input data
+This repository uses GitHub Actions for CI. The workflows are defined in `.github/workflows/`.
 
-Input sequences and metadata can be retrieved from data.nextstrain.org
-
-* [sequences.fasta.xz](https://data.nextstrain.org/files/workflows/monkeypox/sequences.fasta.xz)
-* [metadata.tsv.gz](https://data.nextstrain.org/files/workflows/monkeypox/metadata.tsv.gz)
-
-Note that these data are generously shared by many labs around the world.
-If you analyze and plan to publish using these data, please contact these labs first.
-
-Within the analysis pipeline, these data are fetched from data.nextstrain.org and written to `data/` with:
-
-```bash
-nextstrain build . data/sequences.fasta data/metadata.tsv
-```
-
-### Run analysis pipeline
-
-Run pipeline to produce the "overview" tree for `/mpox/all-clades` with:
-
-```bash
-nextstrain build . --configfile config/mpxv/config.yaml
-```
+## Development
 
-Run pipeline to produce the "clade IIb" tree for `/mpox/clade-IIb` with:
+### Pre-commit
 
-```bash
-nextstrain build . --configfile config/hmpxv1/config.yaml
-```
+This repository uses [pre-commit](https://pre-commit.com/) to run checks on the code before committing.
 
-Run pipeline to produce the "lineage B.1" tree for `/mpox/lineage-B.1` with:
+To install pre-commit on macOS, run:
 
 ```bash
-nextstrain build . --configfile config/hmpxv1_big/config.yaml
+brew install pre-commit
 ```
```

Comment:
Is this a requirement for anyone working on this repo? What happens if I don't install pre-commit and push something to `master` that would fail checks?

Reply:
It's just like snakefmt lint checks, nothing happens other than that pre-commit checks fail and the commit gets a red ❌